### DISCUSSION TITLE
Temario I2

### DISCUSSION THREAD

**Discussion by IgnacioHqz** [IIC2133-PUC/2024-2#169] [Preguntas Materia] [answered]
Hola! Sé que a lo mejor aún es muy temprano para preguntar sobre el temario de la i2 pero se sabe más o menos hasta qué clase entrará? Si me pudieran comentar lo agradecería mucho para priorizar qué estudiar. Muchas gracias!

**Comment by tomastrivino** [comment, marked as answer]
Hola!
El temario será informado por Canvas

Saludos
